test/pattern/comp.c

```diff
@@ -267,7 +267,7 @@ void test_mutt_pattern_comp(void)
       TEST_MSG("Actual  : pat == NULL");
     }
 
-    struct PatternList expected;
+    struct PatternList expected = SLIST_HEAD_INITIALIZER(expected);
     SLIST_INIT(&expected);
```

Comment:
We've got several instances of `SLIST_HEAD_INITIALIZER()` **and** `SLIST_INIT()`.

Reply:
Oh, one of them was missing, it just crashes on me, and I punished them all :/ will review, thanks!